scripts/revng

```diff
@@ -327,16 +327,22 @@ def run_translate(args):
                       "--skip",
                       action="store_true",
                       help="Do not invoke revng-lift.")
+  parser.add_argument("-l",
+                      "--link-only",
+                      action="store_true",
+                      help="Invoke linker only.")
   parser.add_argument("-i",
                       "--isolate",
                       action="store_true",
                       help="Enable function isolation.")
   parser.add_argument("--base", help="Load address to employ in lifting.")
   parser.add_argument("-o", "--output", metavar="OUTPUT", help="Output path.")
   parser.add_argument("input", metavar="INPUT", help="The input binary.")
+  parser.add_argument("--opt", type=str, help="Pass to LLVM opt the passes to be performed over the IR.")
```

Comment:
We will need something more flexible than what we have now, but `--opt` is kind of an hack.
Let's just have another argument such as `--isolate` that does The Right Thing™ for you, as opposed to add passes by hand to `--opt`.

Reply:
For sure specifying every single argument may be more precise, but does `--opt` really look like a hack? To be honest it just seems like kind of a compact way to directly specify the optimizations (e.g., `--opt=isolate,enforce-abi,debug-info`). Maybe it's not well structured right now, but at least is faster than invoking `opt` manually. About to change it though.

Reply:
We have a project in mind for having a declarative way to define meta-commands such as `revng translate`, but not today.
`--opt` is adding an extra layer on an already bad design. I don't want to go that direction.

Reply:
Alright, sure.